src/network/networkd-link.c

```diff
@@ -318,6 +318,19 @@ static int link_enable_ipv6(Link *link) {
         return 0;
 }
 
+static bool link_is_enslaved(Link *link) {
+        if (!link->enslaved)
```

Comment:
hmm, so there's now a function is_enslaved and a boolean "enslaved", and they are kinda the same but also not. I wonder if one of them should have a different name. Maybe the bool should be called "enslaved_raw" or so? (better names might be thinkable)